frontend/src/metabase/account/notifications/components/NotificationCard.jsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+import {
+  formatDay,
+  formatFrame,
+  formatHourAMPM,
+  parseTimestamp,
+} from "metabase/lib/time";
+import * as Urls from "metabase/lib/urls";
+import {
+  NotificationContent,
+  NotificationDescription,
+  NotificationItemRoot,
+  NotificationTitle,
+} from "./NotificationCard.styled";
+
+const propTypes = {
+  item: PropTypes.object.isRequired,
+  type: PropTypes.oneOf(["pulse", "alert"]).isRequired,
+  user: PropTypes.object,
+};
+
+const NotificationCard = ({ item, type, user }) => {
+  return (
+    <NotificationItemRoot>
+      <NotificationContent>
+        <NotificationTitle to={formatLink(item, type)}>
+          {formatTitle(item, type)}
+        </NotificationTitle>
+        <NotificationDescription>
+          {formatDescription(item, user)}
+        </NotificationDescription>
+      </NotificationContent>
+    </NotificationItemRoot>
+  );
+};
+
+NotificationCard.propTypes = propTypes;
+
+const formatTitle = (item, type) => {
+  switch (type) {
+    case "pulse":
+      return item.name;
+    case "alert":
+      return item.card.name;
+  }
+};
+
+const formatLink = (item, type) => {
+  switch (type) {
+    case "pulse":
+      return Urls.dashboard({ id: item.dashboard_id });
+    case "alert":
+      return Urls.question(item.card);
+  }
+};
+
+const formatDescription = (item, user) => {
+  const parts = [
+    ...item.channels.map(formatChannel),
+    formatCreator(item, user),
+  ];
+
+  return parts.join(" · ");
+};
+
+const formatChannel = channel => {
+  let scheduleString = "";
+
+  switch (channel.channel_type) {
+    case "email":
+      scheduleString += t`Emailed `;
+      break;
+    case "slack":
+      scheduleString += t`Slack’d `;
+      break;
+    default:
+      scheduleString += t`Sent`;
+      break;
+  }
+
+  switch (channel.schedule_type) {
+    case "hourly":
+      scheduleString += t`hourly`;
+      break;
+    case "daily": {
+      const ampm = formatHourAMPM(channel.schedule_hour);
+      scheduleString += t`daily at ${ampm}`;
+      break;
+    }
+    case "weekly": {
+      const ampm = formatHourAMPM(channel.schedule_hour);
+      const day = formatDay(channel.schedule_day);
+      scheduleString += t`${day} at ${ampm}`;
+      break;
+    }
+    case "monthly": {
+      const ampm = formatHourAMPM(channel.schedule_hour);
+      const day = formatDay(channel.schedule_day);
+      const frame = formatFrame(channel.schedule_frame);
+      scheduleString += t`monthly on the ${frame} ${day} at ${ampm}`;
+      break;
+    }
+  }
```

Comment:
We should respect users' preferred date-time format (can be selected at `admin/settings/localization`. TBH, it looks like it's mostly ignored, but we should not make it worse at least.

Please take a look at the [`DateTime` component](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/DateTime.jsx) as a possible solution / reference

Reply:
Used `formatDateTimeWithUnit` for formatting a date. It turned out that the function cannot be used to format day of weeks and hours without a date, and new formatting functions should be implemented to replace `formatDay` and `formatHourAMPM` to support localization settings. It doesn't seem hard, but I would like to do that in a follow-up PR. What do you think?

Reply:
Sure! Thank you